cocos/core/math/quat.ts

```diff
@@ -389,6 +414,7 @@ export class Quat extends ValueType {
     }
 
     /**
+     * @en Calculated according to the local orientation quaternion coordinate axis, the default three vectors are normalized and mutually perpendicular
```

Comment:
Calculated the quaternion represents the given coordinates, considering all given vectors are normalized and mutually perpendicular